src/my-bot.ts

```diff
@@ -1,39 +1,72 @@
+import {GameState} from "./gameState"
+import {Position} from "./position"
+import {Probability} from "./probability"
+
 export class MyBot {
     public getShipPositions() {
-        return [
-            { StartingSquare: { Row: "A", Column: 1 }, EndingSquare : { Row: "A", Column: 5 } },
-            { StartingSquare: { Row: "C", Column: 1 }, EndingSquare : { Row: "C", Column: 4 } },
-            { StartingSquare: { Row: "E", Column: 1 }, EndingSquare : { Row: "E", Column: 3 } },
-            { StartingSquare: { Row: "G", Column: 1 }, EndingSquare : { Row: "G", Column: 3 } },
-            { StartingSquare: { Row: "I", Column: 1 }, EndingSquare : { Row: "I", Column: 2 } },
-        ]
+        try {
+            return Probability.getRandomShipPositions();
+        }
+        catch(error){
+            return [
+                { StartingSquare: { Row: "A", Column: 1 }, EndingSquare : { Row: "A", Column: 5 } },
+                { StartingSquare: { Row: "C", Column: 1 }, EndingSquare : { Row: "C", Column: 4 } },
+                { StartingSquare: { Row: "E", Column: 1 }, EndingSquare : { Row: "E", Column: 3 } },
+                { StartingSquare: { Row: "G", Column: 1 }, EndingSquare : { Row: "G", Column: 3 } },
+                { StartingSquare: { Row: "I", Column: 1 }, EndingSquare : { Row: "I", Column: 2 } },
+            ]
+        }
     }
 
-    // gamestate is the body of the post request
+<<<<<<< HEAD
+    // gamestate is the body of the post request i.e. req.body
+    public selectTarget(gamestate:GameState) {
+
+        let previousShot = (gamestate.MyShots.length != 0)?(gamestate.MyShots[gamestate.MyShots.length-1].Position):null;
+=======

```

Comment:
I don't think this stuff should be here - in fact, it should be breaking your app! Did you do a merge before pushing out this review?